tests/testthat/test.wrapper.R

```diff
@@ -202,6 +202,90 @@ test_that("concepts_map no equivalent", {
 })
 
 
+
+# concepts_included_in ----------------------------------------------------
+
+test_that("concepts_included_in", {
```

Comment:
if one were to aim for a complete set of tests, I guess it might be helpful to test the validation of inputs too (as in empty ecl) and also include a test for the progress bar progress; I might have not seen the empty ecl though ...

Reply:
https://github.com/ramses-antibiotics/snomedizer/pull/21/commits/daf8553d6125e07308542a088bbd4e6640bb4296